### ISSUE TITLE
Abnormal mandibular branch of trigeminal nerve

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5903]
**Preferred term label:** Abnormal mandibular branch of trigeminal nerve

**Synonyms** 

**Definition (free text, please give PubMed ID)** Physiological or morphological abnormality of the mandibular branch of the trigeminal nerve, resulting in abnormal jaw movement and/or facial sensation in the lower third of the face., PMID:27025245

**Parent term (use hpo.jax.org/app)** Abnormal fifth cranial nerve physiology HP:0031911

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
We would not add a general (Physiology plus Morphology) term this far down the hierarchy. Could you please make new requests of there are specific morphological or physiological abnormalities?
